app/main.py

```diff
@@ -1,5 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    """
+    """`
```

Comment:
There is no need to add this change here, remove it, please

Reply:
I don`t underdtand, what change?